tests/Resources/ti.network.httpclient.addontest.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2019 Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe('Titanium.Network.HTTPClient', function() {
+  this.timeout(6e4);
+
+  it.android('TLSv3 support', function(finish) {
+    // Only supported on Android 10+
+    if (parseInt(Ti.Platform.version.split('.')[0]) < 10) {
+      return finish();
+    }
+
+    const client = Ti.Network.createHTTPClient({
+      onload : e => {
+        if (html.includes('protocol_ssl3">Yes')) {
+          finish();
+        }
+      },
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.network.httpclient.addontest.js;
  Line: 26;
-->
- :no_entry_sign: tests/Resources/ti.network.httpclient.addontest.js line 26 – Expected indentation of 3 tabs but found 6 spaces. (indent)